resources/css/completion-progress.css

```diff
@@ -92,3 +92,49 @@
     @apply text-2xs flex justify-between w-full;
   }
 }
+
+/* 
+ * Completion indicators on progress bars also use a lot of duplicative class names
+ * which can heavily bulk up the document size.
+*/
+
+/** game-progress-bar.blade.php **/
+.gprogress-ind__root {
+  @apply cursor-help absolute right-0 -top-2 rounded-full border-2 w-5 h-5 z-20;
+}
+
+.gprogress-ind__root[data-award='unfinished'] {
+  @apply border-embed-highlight bg-embed;
+}
+
+.gprogress-ind__root[data-award='beaten-softcore'] {
+  @apply border-zinc-400 light:border-zinc-500 bg-embed;
+
+  > div {
+    @apply bg-zinc-600/50 blur-[8px] light:bg-zinc-400/90 w-full h-full z-20;
+  }
+}
+
+.gprogress-ind__root[data-award='beaten-hardcore'] {
+  @apply border-zinc-600 bg-gradient-to-br from-zinc-300 to-zinc-500 light:to-zinc-100;
+
+  > div {
+    @apply bg-zinc-600/50 blur-[4px] h-full;
+  }
+}
+
+.gprogress-ind__root[data-award='completed'] {
+  @apply border-yellow-500 light:border-yellow-500 bg-embed;
+
+  > div {
+    @apply bg-yellow-700/50 blur-[8px] light:bg-yellow-600/40 w-full h-full z-20;
```

Comment:
z-order is wrong
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ed1ee228-dff0-45ea-b87b-3ee406eaf077)
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/b6176a3d-acd7-4726-bf2d-b365beeed38b)

Reply:
Thanks, resolved in latest.